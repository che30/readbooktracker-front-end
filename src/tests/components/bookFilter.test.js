import React from 'react';
import renderer from 'react-test-renderer';
import BookFilter from '../../components/bookFilter';

it('renders properly', () => {
  const tree = renderer
    .create(<BookFilter />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
