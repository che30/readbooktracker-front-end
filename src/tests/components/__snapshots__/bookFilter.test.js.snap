// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<div>
  Fetching data
</div>
`;
